adoc/admin-storage-vsphere.adoc

```diff
@@ -0,0 +1,248 @@
+= vSphere Storage
+The vSphere cloud provider can be enabled with {productname} to allow Kubernetes pods to use VMWare vSphere Virtual Machine Disk (VMDK) volumes as persistent storage.
```

Comment:
Maybe it deserves a link to refer to the deployment guide of the vSphere VCP.

Reply:
done